mysqld_exporter.go

```diff
@@ -154,6 +160,16 @@ var landingPage = []byte(`<html>
 
 // Metric descriptors for dynamically created metrics.
 var (
+	binlogSizeDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, binlog, "size"),
```

Comment:
But if this is a size gauge in bytes, add a unit suffix like `..._binlog_size_bytes`. If it's just a current count of something (like the `..._binlog_files` metric below, no suffix is needed).